plugins/csv-adapter/src/main/java/org/polypheny/db/adapter/csv/CsvFieldType.java

```diff
@@ -34,21 +34,21 @@
 package org.polypheny.db.adapter.csv;
 
 
+import java.util.HashMap;
+import java.util.Map;
 import org.apache.calcite.linq4j.tree.Primitive;
 import org.polypheny.db.adapter.java.JavaTypeFactory;
 import org.polypheny.db.algebra.type.AlgDataType;
+import org.polypheny.db.catalog.exceptions.GenericRuntimeException;
 import org.polypheny.db.type.PolyType;
 
-import java.util.HashMap;
-import java.util.Map;
-
 
 /**
  * Type of a field in a CSV file.
  *
  * Usually, and unless specified explicitly in the header row, a field is of type {@link #STRING}. But specifying the field type in the header row makes it easier to write SQL.
  */
-enum CsvFieldType {
+public enum CsvFieldType {
     STRING( String.class, "string" ),
```

Comment:
Why not directly specify the new PolyTypes? First using Java types seems unnecessary complex.

Reply:
Artifact, refactored class to use `PolyValues`.